mindpype/graph.py

```diff
@@ -354,9 +354,9 @@ def initialize(self, default_init_data=None, default_init_labels=None):
                                                             sys.exc_info()[2])
 
         self._initialized = True
-        self.session.free_unreferenced_data()
+        self.session._free_unreferenced_data()
 
-    def update(self):
+    def _update(self):
```

Comment:
This was added after I gave you the API list. This should also be included as a public method (same for the corresponding methods within Graph and Kernel)